Wikipedia/Code/LanguageCell.m

```diff
@@ -19,6 +20,8 @@ @interface LanguageCell ()
 @property (strong, nonatomic) IBOutlet UILabel* languageCodeLabel;
 
 @property (strong, nonatomic) IBOutlet NSLayoutConstraint* languageNameLabelHeight;
+@property (strong, nonatomic) IBOutlet UIView* separatorView;
```

Comment:
We should use built in UITableViewCell separator APIs unless there is something specific from the designs that we aren't able to do otherwise.

Did you run into a limitation that prevented you from using them?